src/static/css/page.css

```diff
@@ -295,6 +296,46 @@ img {
   max-width: 100%;
   height: auto;
 }
+
+article h2 a,
+article h3 a,
```

Comment:
There's a lot of repetition. Could we target the anchors more directly using a class selector, so we don't need the ancestry tree?

Reply:
Good idea - can add the class when I add the link.

Reply:
Fixed!